app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ReservationModel.java

```diff
@@ -0,0 +1,57 @@
+package com.mobile.tandil.javabaseproject.mvp.model;
+
+import com.mobile.tandil.javabaseproject.database.ReservationDatabase;
+import com.mobile.tandil.javabaseproject.entity.Reservation;
+import com.mobile.tandil.javabaseproject.mvp.contract.ReservationContract;
+import com.mobile.tandil.javabaseproject.utils.Constants;
+import com.mobile.tandil.javabaseproject.utils.ValidationEnums;
+import java.util.Calendar;
+import java.util.List;
+
+public class ReservationModel implements ReservationContract.ReservationModel {
+
+    private ReservationDatabase database;
+
+    public ReservationModel(ReservationDatabase database) {
+        this.database = database;
+    }
+
+    @Override
+    public void addReservation(Reservation reservation) {
+        database.addReservation(reservation);
+    }
+
+    private boolean ReservationAlreadyExists(Calendar checkIn, Calendar checkOut, int parking) {
```

Comment:
this method name should be in cammelCase

Reply:
Done